src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -3083,10 +3083,8 @@ public IAsyncResult BeginInvoke<T>(PSDataCollection<T> input, PSInvocationSettin
         /// <exception cref="ObjectDisposedException">
         /// Object is disposed.
         /// </exception>
-        public async Task<PSDataCollection<PSObject>> InvokeAsync()
-        {
-            return await Task<PSDataCollection<PSObject>>.Factory.FromAsync(BeginInvoke(), _endInvokeMethod).ConfigureAwait(false);
-        }
+        public Task<PSDataCollection<PSObject>> InvokeAsync()
```

Comment:
Can we use ValueTask?

Reply:
Quoted from [its doc](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=netcore-2.2#remarks)

> A method may return an instance of this value type when it's likely that the result of its operation will be available synchronously, and when it's expected to be invoked so frequently that the cost of allocating a new Task<TResult> for each call will be prohibitive.

I don't think `ValueTask` fits here.

Reply:
Thanks! I did not found any scenario too.